src/citrine/resources/team.py

```diff
@@ -165,16 +217,21 @@ def update_user_action(self, user_id: Union[str, UUID], *,
             Returns ``True`` if user successfully added
 
         """
+        if isinstance(user_id, User):
+            user_id = user_id.uid
         self.session.checked_put(self._path() + "/users", version=self._api_version,
                                  json={'id': str(user_id), "actions": actions})
         return True
 
-    def share(self, *,
+    def share(self,
+              *,
               resource: Resource,
-              target_team_id: Union[str, UUID]) -> bool:
+              target_team_id: Union[str, UUID, "Team"]) -> bool:
```

Comment:
You document that the user can provide a `Team` object, but I don't see any handling for that input below.

Reply:
Oh, good catch.  I neglected to add the map-team-to-uid conditional on this one.